tests-ng/test_sysctls.py

```diff
@@ -2,6 +2,7 @@
 
 
 @pytest.mark.booted
+@pytest.mark.feature("not cis", reason="CIS needs rp_filter to be strict")
```

Comment:
is there a cis version of this test, to ensure the cis feature that this set correctly?